scripts/simex

```diff
@@ -362,39 +362,43 @@ experiments_launch_parser.add_argument('--queue', type=str)
 def do_experiments_purge(args):
 	cfg = extl.base.config_for_dir()
 
-	for run in select_runs_from_cli(cfg, args, default_all=False):
-		(exp, instance) = (run.experiment, run.instance.filename)
-
-		if args.f:
-			print("Purging experiment '{}', instance '{}' [{}]".format(
-					exp.name, instance, run.repetition))
-			try:
-				os.unlink(run.aux_file_path('lock'))
-			except FileNotFoundError:
-				pass
-			try:
-				os.unlink(run.aux_file_path('run'))
-			except FileNotFoundError:
-				pass
-			try:
-				os.unlink(run.aux_file_path('run.tmp'))
-			except FileNotFoundError:
-				pass
-			try:
-				os.unlink(run.output_file_path('out'))
-			except FileNotFoundError:
-				pass
-			try:
-				os.unlink(run.output_file_path('status'))
-			except FileNotFoundError:
-				pass
-			try:
-				os.unlink(run.output_file_path('status.tmp'))
-			except FileNotFoundError:
-				pass
-		else:
-			print("This would purge experiment '{}', instance '{}' [{}]".format(
-					exp.name, instance, run.repetition))
+	if can_select_runs_from_cli(args):
```

Comment:
This would be much prettier if we just did:
```
if not can_select_runs_from_cli(args):
    print(...)
    return
```

Reply:
I have made the changes and also added a small help message for the '-f' option when calling 'simex e purge -h'.